### ISSUE TITLE
Can you share pelee classification newwork on imagenet?

### ISSUE THREAD

**Issue by ujsyehao** [Robert-JunWang/Pelee#32]
Hi @Robert-JunWang, I want to train pelee classification network, I am confused about solver.prototxt, Can you provide your train file including train.prototxt, test.prototxt and solver.prototxt?
Thank you in advance!

**Comment by Robert-JunWang** [comment]
My PeleeNet model is trained by Pytorch. I will share it later. To train on Caffe, you can manually modify the output of peleenet.py (modifying the data layer and adding the Accuracy and SoftmaxWithLoss layers). 
```
python peleenet.py > deploy_peleenet.prototxt
```

**Comment by ujsyehao** [comment]
Thank you @Robert-JunWang !
I will try to train pelee classification model.

**Comment by ujsyehao** [comment]
Hi, I am sorry to disturb you.
I have deploy_peleenet.prototxt(use peleenet.py) and turn it to peleenet_trainval.prototxt
I am confused about peleenet_inet_acc7243.caffemodel. what is the top-1 accuracy? Is it 72.43?
I use peleenet_inet_acc7243.caffemodel to do test(use peleenet_trainval.prototxt), I get a bad top-1 accuracy. Do you modify peleenet network?

**Comment by zys1994** [comment]
> Hi, I am sorry to disturb you.
> I have deploy_peleenet.prototxt(use peleenet.py) and turn it to peleenet_trainval.prototxt
> I am confused about peleenet_inet_acc7243.caffemodel. what is the top-1 accuracy? Is it 72.43?
> I use peleenet_inet_acc7243.caffemodel to do test(use peleenet_trainval.prototxt), I get a bad top-1 accuracy. Do you modify peleenet network?

can you share your deploy_peleenet.prototxt?  i can't use [python peleenet.py > deploy_peleenet.prototxt] to get it

**Comment by lijingle-coder** [comment]
> Hi, I am sorry to disturb you.
> I have deploy_peleenet.prototxt(use peleenet.py) and turn it to peleenet_trainval.prototxt
> I am confused about peleenet_inet_acc7243.caffemodel. what is the top-1 accuracy? Is it 72.43?
> I use peleenet_inet_acc7243.caffemodel to do test(use peleenet_trainval.prototxt), I get a bad top-1 accuracy. Do you modify peleenet network?

do you train imagenet using peleenet,  the accuracy is 73,or some, I train ImageNet， but I get 64% accuracy? can you share you result@yehao ujsyehao

**Comment by ujsyehao** [comment]
@lijingle-coder I don't train peleenet, I just test it.
